facefusion/face_helper.py

```diff
@@ -121,3 +121,19 @@ def apply_nms(bbox_list : List[Bbox], iou_threshold : float) -> List[int]:
 		iou = width * height / (areas[index] + areas[remain_indices] - width * height)
 		indices = indices[numpy.where(iou <= iou_threshold)[0] + 1]
 	return keep_indices
+
+
+def translate_age(age : int) -> FaceAnalyserAge:
```

Comment:
what about `categorize` or `determine` instead of `translate`

Reply:
done